charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -243,6 +243,10 @@ spec:
             value: "kube_endpoints kube_services"
           - name: DD_EXTRA_LISTENERS
             value: "kube_endpoints kube_services"
+          {{- if .Values.datadog.secretBackend.command -}}
+          - name: DD_SECRET_BACKEND_SKIP_CHECKS
+            value: "true"
+          {{- end }}
```

Comment:
I cluster checks was not decrypting the secrets until I enabled this.

I am not sure why and could not find any documentation except this PR, I think there is missing Documentation about this setting
- https://github.com/DataDog/datadog-agent/pull/8492

Reply:
that's a good catch, we don't have the helm option to use this.
I don't think it should be flipped on from the usage of a command though.
I would recommend adding a new parameter `Values.datadog.secretBackend.skipChecks`.
I can do this as a follow up PR, and for now maybe use:
```
clusterAgent
  env:
      - name: DD_SECRET_BACKEND_SKIP_CHECKS
        value: true
```